docs/alerting/exporter-metrics.md

```diff
@@ -30,3 +30,4 @@ Some important metrics are listed below.
 | `service_net_out_byte`| `gpu_exporter` | Network out traffic by pai service detectived by docker stats (byte) |
 | `service_block_in_byte`| `gpu_exporter` | Block io in traffic by pai service detectived by docker stats (byte) |
 | `service_block_out_byte`| `gpu_exporter` | Block io out traffic by pai service detectived by docker stats (byte) |
+| `docker_daemon_count` | `gpu_exporter` | Count of docker daemon, count is always 1, but error label indicate what type of error this docker daemon has |
```

Comment:
why count is always 1? 1 represent correct state? if this "Count of docker daemon" is wrong?

Reply:
this is similar to watchdog's metrics, use label to indicate error, the value of metrics is useless in single metrics. This way we can use `sum(docker_daemon_count{"error"!="ok"}) > 0` to do alert.

Reply:
Got it, Maybe change to similar explain will be easy understand for user "Used to summary healthy count of docker daemon,  this metrics per server value is always 1, leverage label to filter health metrics and accumulate them to a sum count for watchdog to use ...."